pages/api/announcements/index.ts

```diff
@@ -33,6 +33,7 @@ async function sendNotifications(announcement: any) {
       data: {
         ...announcement,
         baseUrl: process.env.BASE_URL,
+        iconUrl: process.env.NEXT_PUBLIC_ICON_URL,
```

Comment:
Better to hardcode the icon URL, we're hitting the 4KB limit already. Add a !change flag for organizers